port.js

```diff
@@ -97,11 +91,11 @@ module.exports = (defaults) => class Port extends EventEmitter {
             }
             return prev;
         }.bind(this), methods);
-        return this.bus && Promise.all([
+        await this.bus && Promise.all([
             this.bus.register(methods.req, 'ports', this.config.id),
-            this.bus.subscribe(methods.pub, 'ports', this.config.id),
-            this.bus && typeof this.bus.portEvent instanceof Function && this.bus.portEvent('init', this)
+            this.bus.subscribe(methods.pub, 'ports', this.config.id)
         ]);
+        return this.fireEvent('init');
```

Comment:
Maybe we can remove this line, as currently all module methods are attached in ports' start methods and the init events are not fired anyway